core/templates/components/button-directives/create-activity-button.component.html

```diff
@@ -1,17 +1,17 @@
-<div class="oppia-navbar-button-container">
+<div class="oppia-navbar-button-container btn-group">
```

Comment:
Is this required?

Reply:
Although  the uplaod functionality is not complete yet but the buttons looks like this without this change
![image](https://user-images.githubusercontent.com/60519303/113856184-896af280-97be-11eb-8892-6d5bf7392e74.png)

Reply:
IIRC the upload button is hidden behind a flag. Can you please check?

Reply:
Ah I see. sg, thanks!